server/services/reportService.ts

```diff
@@ -41,6 +43,7 @@ const toReport = (namesByOffenderNumber: NamesByOffenderNumber) => (
 export default class ReportService {
   constructor(
     private readonly incidentClient: IncidentClient,
+    private readonly draftReportClient: DraftReportClient,
```

Comment:
this is no longer used so can be removed

Reply:
have done